hack/verify-manifests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+manifest_file=config/crd/mcad.yaml
+appwrappers=appwrappers.workload.codeflare.dev
+quotasubtrees=quotasubtrees.quota.codeflare.dev
+schedulingspecs=schedulingspecs.workload.codeflare.dev
+
+echo $manifest_file
+if grep $appwrappers $manifest_file \
```

Comment:
We would rather want to check there is no diff, than simply check the files are present.